apps/src/templates/sectionProgress/standards/StandardsIntroDialog.jsx

```diff
@@ -21,17 +23,32 @@ const styles = {
   }
 };
 
+/*
+Dialog that show the first time a teacher goes to the
+Standards view of the Progress Tab in Teacher Dashboard
+ */
+
 class StandardsIntroDialog extends Component {
   static propTypes = {
     isOpen: PropTypes.bool.isRequired,
-    handleConfirm: PropTypes.func.isRequired
+    setCurrentUserHasSeenStandardsReportInfo: PropTypes.func.isRequired
+  };
+
+  dismissStandardsDialog = () => {
+    $.ajax({
+      url: '/dashboardapi/v1/users/me/set_standards_report_info_to_seen',
+      type: 'post',
+      data: {}
+    }).done(() => {
+      this.props.setCurrentUserHasSeenStandardsReportInfo(true);
```

Comment:
Is there any way for this post to fail? If so, should the error case be handled?

Reply:
This is a good thing to think about, but if the worst case is that we show the dialog again, then I think it's fine not to handle the error case.

Reply:
I don't know I guess it could possibly fail. If it doesn't post it just seems like it would mean that the dialog would not close right? So maybe on fail we should still close the dialog... they might see it again next time they reload the page but that might be ok?